sim/town-generator.ipynb

```diff
@@ -164,8 +144,8 @@
     "    # proportion of total population that are essential workers\n",
     "    essential_to_total_ratio = 0.2\n",
     "    \n",
-    "    # which worker types to include (0:education, 1:office, 2:social, 3:supermarket)\n",
-    "    incl_worker_types = [0,1,2,3]\n",
+    "    # which worker types to include (0:education, 1:office, 2:retail stores, 3:social, 4:supermarket)\n",
```

Comment:
On my end this is the order of site types after reading in query files.

Reply:
Yes, this is good, I would like to keep it alphabetical, plus home gatherings at the end.